src/Sylius/Bundle/ShopBundle/Resources/views/Product/Show/_price.html.twig

```diff
@@ -1,3 +1,3 @@
 <span class="ui huge header" id="product-price">
-    {{ product.price|sylius_price }}
+    {{ product.variants.first()|sylius_channel_variant_price }}
```

Comment:
Yup, but it would require to implement separate filter - which would make more mess in this already messed up PR 😄  For now I would vote to leave it as it is (moreover, this has been done this way before -> by getting price with ``getPrice()`` method) and improve it later.